jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -98,8 +98,41 @@ public static boolean getDefaultUseCaches()
         return __defaultUseCaches;
     }
 
+    public static Resource.Mount mountCollection(Collection<Resource> resources)
```

Comment:
That looks weird: you should not be able to create a Resource that needs mounting. I think this method should return `Resource`, be called `newResourceCollection` and assume everything is properly mounted.

Reply:
Method removed